lib/unleash/variant_definition.rb

```diff
@@ -2,8 +2,19 @@
 
 module Unleash
   class VariantDefinition
+    include Comparable
     attr_accessor :name, :weight, :payload, :overrides, :stickiness
 
+    def ==(other)
+      equal = true
+      equal = false unless self.name == other.name
+      equal = false unless self.weight == other.weight
+      equal = false unless self.payload == other.payload
+      equal = false unless self.stickiness == other.stickiness
+      equal = false unless self.overrides == other.overrides
+      equal
```

Comment:
Bikeshedding for improved style:
```suggestion
      self.name == other.name &&
      self.weight == other.weight &&
      self.payload == other.payload &&
      self.stickiness == other.stickiness &&
      self.overrides == other.overrides
```